docs/BR.md

```diff
@@ -1316,25 +1324,20 @@ defined by RFC6960.
 
 OCSP responders operated by the CA SHALL support the HTTP GET method, as described in RFC 6960 and/or RFC 5019.
 
-The validity interval of an OCSP response is the difference in time between the `thisUpdate` and `nextUpdate` field, inclusive. For purposes of computing differences, a difference of 3,600 seconds shall be equal to one hour, and a difference of 86,400 seconds shall be equal to one day, ignoring leap-seconds.
+For the purpose of computing an OCSP Validity Interval, a difference of 3,600 seconds shall be equal to one hour, and a difference of 86,400 seconds shall be equal to one day, ignoring leap-seconds.
```

Comment:
```suggestion
As stated in [Section 1.6.4](#164-conventions), for the purpose of computing an OCSP Validity Interval, an hour is measured as 3,600 seconds and a day is measured as 86,400 seconds, ignoring leap seconds.
```

Reply:
Shouldn't this section also make the reference to section 1.6.4 effective 6-1-2022?

Reply:
I was of two minds on that. It doesn't *have* to, because causing the BRs to ignore leap seconds for all other time computations is actually _maintaining the status quo_ for OCSP validity intervals. So there's technically no need to have an effective date in this section.

On the other hand, the lack of symmetry with the other sections really bothers me, yeah.